docs/guides/postman-collection.md

```diff
@@ -0,0 +1,87 @@
+# Postman Collections support
+
+Prism offers a _limited_ support for Postman Collection. The basic workflow of using Prism (both from the CLI and deployed in a Docker container) are fundamentally the same, however there are some known limitations that it's important to keep in mind:
+
+## Authentication
+
+Postman collections support more authentication methods than OpenAPI 2 and 3, although some of these are a specialisation of another method included in their support; this is mostly to drive the UI of their application.
+
+In case the authentication method does not have a corresponding element in the OpenAPI specification, it can ultimately be expressed as a header. In such a case, Prism will "downgrade" according to the following table:
```

Comment:
I understand that in http-spec we convert hawk to hardcoded headers but what does that mean for Prism, and specifically what does "downgrade" mean for Security Validation? 

What would the user expect to happen, or what can the user do to make it better? Does security validation simply not work for these authentication types? Will Prism emit a warning and just skip the security validation on request or will it error on load?

Reply:
Good questions, I should have clarified a little bit better:

It fundamentally means that Prism will only verify the header presence, but it will completely skip validate the single parameters that are used to construct the required hawk header. For example, some of them are mandatory, some of them are not — we won't do such check. If there's the required header and it follows a basic patter (https://github.com/stoplightio/http-spec/blob/master/src/postman/transformers/securityScheme.ts#L159) we'll mark the security validation as passed, although the header might be hawk invalid.

Does that clarify? If so, I can update the document with this explanation.

Reply:
Great. We could reduce this entire section to "Postman supports a few authentication schemes which OpenAPI does not, such as Hawk and AWS. In this case Prism will do a simplified version of security validation, and just check you've got the right headers populated." or something like that. We don't need to get too in depth.

Reply:
Make sense. I'll simplify